src/librustdoc/html/static/styles/main.css

```diff
@@ -26,9 +26,6 @@ h1.fqn {
 h2, h3:not(.impl):not(.method):not(.type):not(.tymethod), h4:not(.method):not(.type):not(.tymethod) {
 	border-bottom-color: #DDDDDD;
 }
-.in-band {
-	background-color: white;
-}
```

Comment:
I made a change that conserves the .in-band style here: [https://github.com/rust-lang/rust/pull/43979/commits/4729f22f8b3cc9faaf147909c6fbec3f9e434d4c](url)